packages/samples/mvp-dapp/src/app.controller.ts

```diff
@@ -60,7 +60,7 @@ router.get<never, { path: string; address: string }>('/load/:address/:path', asy
   )
   const value = recordModel.load(`data.${params.path}`)
   if (value) {
-    ctx.ok(MvpResponse.ok(MvpResponse.ok(value)))
+    ctx.ok(MvpResponse.ok(value))
```

Comment:
So the nested response is caused by business code, and the response is not wrapped by `kuai` automatically? 🤔

Maybe we can schedule the automatic wrapping of responses at the most outside as 

```mermaid
flowchart TD
	S[Service]
	C[Controller]
	I[Interceptor]
  W[Wrapped response]
  S -- res --> C -- res --> I --> W
  S -- res --> I --> W
```